docs/api.md

```diff
@@ -1535,6 +1536,27 @@ const iPhone = puppeteer.devices['iPhone 6'];
 
 List of all available devices is available in the source code: [src/common/DeviceDescriptors.ts](https://github.com/puppeteer/puppeteer/blob/main/src/common/DeviceDescriptors.ts).
 
+#### page.emulateCPUThrottling(rate)
```

Comment:
needs updating here too?

Reply:
Yeah, you're absolutely right. Forgot to save the changes so git did get them -.-